spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -67,6 +63,12 @@ fun calculatePackageVerificationCode(files: List<File>) =
             file.inputStream().use { DigestUtils.sha1Hex(it) }
         })
 
+/**
+ * A Kotlin-style convenience function to replace EnumSet.of() and EnumSet.noneOf().
+ */
+inline fun <reified T : Enum<T>> enumSetOf(vararg elems: T): EnumSet<T> =
+        EnumSet.noneOf(T::class.java).apply { addAll(elems) }
+
```

Comment:
That function should be next to `EnumSet.plus` in `Extensions.kt`.

Reply:
Why? It's not an extension function after all.